pageserver/src/tenant.rs

```diff
@@ -1794,7 +1794,11 @@ impl Tenant {
             .map_err(LoadLocalTimelineError::ResumeDeletion)?;
         }
         let needs_manifest_upload =
-            offloaded_timelines_list.len() != preload.tenant_manifest.offloaded_timelines.len();
+            // Upload a new manifest if the stripe size changes. This typically happens when the
```

Comment:
This will generate a bunch of uploads after we deploy -- I think that's okay though, as we already concurrency-limit how many attaches we do concurrently

Reply:
Yeah, I think it should be fine -- attaches are already kind of expensive, and we'd presumably do something similar if we had to bump generation numbers.